lib/faker/cat/cat.ex

```diff
@@ -2,25 +2,177 @@ defmodule Faker.Cat.En do
   import Faker, only: [sampler: 2]
 
   @moduledoc """
-  Functions for Cat names, breeds and registries  in English
+  Functions for Cat names, breeds and registries in English
   """
 
   @doc """
-  Returns a Cat name
+  Returns a Cat name string
+
+  ## Examples
+
+      iex> Faker.Cat.En.name()
```

Comment:
Interesting. The file should actually be `lib/faker/cat/en.ex`. Would you mind to rename it?